pkg/routeagent-driver/handlers/kubeproxy-iptables/kp-iptables.go

```diff
@@ -66,8 +66,8 @@ func (kp *SyncHandler) GetName() string {
 	return "kubeproxy-iptables-handler"
 }
 
-func (kp *SyncHandler) GetNetworkPlugin() string {
-	return event.AnyNetworkPlugin
+func (kp *SyncHandler) GetNetworkPlugins() []string {
+	return []string{"generic", "canal-flannel", "weave-net", "OpenShiftSDN"}
```

Comment:
"flannel"?

Reply:
We need to use what our discovery mechanism provides today in the operator, which is canal-flannel

https://github.com/submariner-io/submariner-operator/blob/master/pkg/discovery/network/canal.go#L51

In an ideal world we would fetch the constants from there, but... we cannot make submariner depend on the operator.

We could define constants here and use them from the operator instead.

Reply:
We could also define such constants in admiral.